conf/parsers_openstack.conf

```diff
@@ -0,0 +1,62 @@
+# Example openstack parsers, see source for recent updates:
+# https://github.com/salt-formulas/salt-formula-fluentbit/tree/master/metadata/service/config/parser
+
+# Take these parsers as an example, as they may not fully fit all possible
+# openstack deployments from all vendors, etc..
+
+[PARSER]
+    # http://rubular.com/r/IvZVElTgNl
+    Name ceph
```

Comment:
Since Ceph is a different product - should we keep it in a separate file?

Reply:
I think so , ok